VP-fitness-frontend/app/src/main/java/com/example/neil/vp_fitness_frontend/OnboardingFragmentActivity.java

```diff
@@ -0,0 +1,230 @@
+package com.example.neil.vp_fitness_frontend;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.support.v7.app.AppCompatActivity;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import com.avontell.fontutil.FontUtil;
+import com.avontell.pagerindicatorbinder.IndicatorBinder;
+import com.matthewtamlin.sliding_intro_screen_library.indicators.DotIndicator;
+
+import java.util.HashMap;
+
+public class OnboardingFragmentActivity extends AppCompatActivity {
+
+    /**
+     * The {@link android.support.v4.view.PagerAdapter} that will provide
+     * fragments for each of the sections. We use a
+     * {@link FragmentPagerAdapter} derivative, which will keep every
+     * loaded fragment in memory. If this becomes too memory intensive, it
+     * may be best to switch to a
+     * {@link android.support.v4.app.FragmentStatePagerAdapter}.
+     */
+    private SectionsPagerAdapter mSectionsPagerAdapter;
+
+    /**
+     * Context for this activity
+     */
+    private Context context;
+
+    /**
+     * Mapping for all onboarding screens
+     */
+    HashMap<String, Fragment> fragments = new HashMap<>();
+
+    /**
+     * The {@link ViewPager} that will host the section contents.
+     */
+    private ViewPager mViewPager;
```

Comment:
Get rid of the default Javadoc for this variable, and instead include a more useful comment about it's purpose.